katsdpimager/loader_ms.py

```diff
@@ -309,6 +327,38 @@ def polarizations(self):
     def has_feed_angles(self):
         return self._feed_angle_correction
 
+    def extra_fits_headers(self):
+        headers = astropy.io.fits.Header()
+        obsgeo = np.mean(self.antenna_positions().to_value(units.m), axis=0)
```

Comment:
Why use the mean antenna position? Does MS lack an explicit reference antenna?

Reply:
I'm not aware of any such thing in MSv2.